core/store/src/db.rs

```diff
@@ -226,6 +227,7 @@ impl DBTransaction {
 pub struct RocksDB {
     db: DB,
     cfs: Vec<*const ColumnFamily>,
+    _pin: PhantomPinned,
```

Comment:
What's was the intention of pining related changes here (adding `PhantomPinned` and `Pin<dyn ...>`)? On the first glance, it seems as if they don't do anything useful here.

Reply:
cc @bowenwang1996 

I'm simply worried that we mention such change the second time in two weeks. First was https://github.com/near/nearcore/pull/5722.

Reply:
I am not sure 😅